packages/eslint-config-typescript/package.json

```diff
@@ -25,9 +25,12 @@
   ],
   "main": "index.js",
   "dependencies": {
-    "@madebywild/eslint-config-base": "^0.2.3"
+    "@madebywild/eslint-config-base": "^0.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^3.4.0"
```

Comment:
same of https://github.com/madebywild/eslint-config/pull/3/files/161a990a67f655b5b89e26095599d29aa3e882bb..49858376ee32a25e390dbfa58bb3fa5838a3d467#r685208094

Reply:
I know, but that doesn't work, that's why it's in both spots